build.gradle

```diff
@@ -10,7 +10,7 @@ repositories {
 	mavenCentral()
 }
 
-def runeLiteVersion = 'latest.release'
+def runeLiteVersion = '1.10.24-SNAPSHOT'
```

Comment:
This should be reverted to `latest.release` as the snapshot is no longer necessary